cropper/src/main/java/com/canhub/cropper/CropImage.kt

```diff
@@ -157,6 +157,7 @@ object CropImage {
      * @param context used to access Android APIs, like content resolve, it is your
      * activity/fragment/widget.
      */
+    @JvmStatic
     fun getPickImageChooserIntent(context: Context): Intent {
```

Comment:
It is possible we add this method usage on the Java sample code? Just wanna be sure no one change it and break in the future =)

Reply:
I am not sure how you want this done within the current sample code.  I think a better approach would be for you to add it to the Kotlin sample code in a way that it can also be added to the Java sample code.  That said, I am not sure that it is important to have both variants of  getPickImageChooserIntent in the sample code.